third_party/libunwindstack/PeCoffInterface.cpp

```diff
@@ -252,6 +261,113 @@ bool PeCoffInterface<AddressType>::ParseOptionalHeader(uint64_t offset) {
     }
   }
 
+  coff_memory_.set_cur_offset(end_offset);
+
+  return true;
+}
+
+template <typename AddressType>
+bool PeCoffInterface<AddressType>::ParseSectionHeaders(uint64_t offset) {
+  uint16_t num_sections = coff_header_.nsects;
+
+  coff_memory_.set_cur_offset(offset);
+
+  for (uint16_t idx = 0; idx < num_sections; ++idx) {
+    SectionHeader section_header;
+
+    // Section names in the header are always exactly 8 bytes. Longer names have to be looked
+    // up in the string table.
+    if (!coff_memory_.GetFully(static_cast<void*>(&section_header.name[0]), 8)) {
```

Comment:
The `8` is used in a lot of different places, especially in the tests. Readability would benefit from it being a constant.

Reply:
Good call, done.